Src/HALAL/Services/Encoder/Encoder.cpp

```diff
@@ -49,40 +43,44 @@ void Encoder::turn_on_encoder(uint8_t id){
 	}
 }
 
-void Encoder::turn_off_encoder(uint8_t id){
-	if (!Encoder::service_IDs.contains(id))
-		return;
+void Encoder::turn_off(uint8_t id){
+	if (not Encoder::service_ids.contains(id)) {
+		return; // TODO error handler
+	}
 
-	TIM_HandleTypeDef* timer =  Encoder::pin_timer_map[service_IDs[id]];
+	TIM_HandleTypeDef* timer = pin_timer_map[service_ids[id]];
 
-	if (HAL_TIM_Encoder_Stop(timer, TIM_CHANNEL_ALL) != HAL_OK	) {
-		//TODO: Exception handle, Warning (Error stopping encoder)
+	if (HAL_TIM_Encoder_Stop(timer, TIM_CHANNEL_ALL) != HAL_OK) {
+		// TODO: Exception handle, Warning (Error stopping encoder)
```

Comment:
Los TODO: van juntos al comentario no le dejes espacio.
```suggestion
		//TODO: Exception handle, Warning (Error stopping encoder)
```